### ISSUE TITLE
[Proposal] Support Falcon

### ISSUE THREAD

**Issue by aVariengien** [TransformerLensOrg/TransformerLens#331]
### Proposal 

The [Technology Innovation Institute](https://www.tii.ae/) released a model family called _falcon_. It contains two sizes, 7b and 40b, each of them have an instruct version. It could be cool if they were supported by TransformerLens. To my knowledge, these are the only open-source instruct fine-tuned model currently available. 

The HF models can be found [here](https://huggingface.co/tiiuae).

### Checklist

- [x] I have checked that there is no similar [issue](https://github.com/neelnanda-io/transformerlens/issues) in the repo (**required**)

**Comment by alan-cooney** [comment]
Agreed this would be great! Would you be interested in writing a PR for it?

**Comment by jlarson4** [comment]
@aVariengien Falcon is now supported as of TransformerLens 3.0, as part of the `TransformerBridge` system. All confirmed supported Falcon models are listed [here in the TransformerBridge Model Table](https://transformerlensorg.github.io/TransformerLens/generated/transformer_bridge_models.html?arch=FalconForCausalLM). If you're migrating an existing project to 3.0, there is also a [migration guide](https://transformerlensorg.github.io/TransformerLens/content/migrating_to_v3.html). Let me know if you have any questions!

Thanks!
